versions.tf

```diff
@@ -12,7 +12,7 @@ terraform {
     }
     kubernetes  = {
       source  = "hashicorp/kubernetes"
-      version = "1.13.0" # Constrained by Google
+      version = "2.13.0" # Constrained by Google
```

Comment:
Same here no breaking changes? Going from version 1 -> version 2

Reply:
There was one small change with this update from my notes above:

> Updated hashicorp/kubernetes to 2.13.0
Removed load_config_file from the provider parameters, that feature was deprecated in hashicorp/kubernetes v2 and we were setting that value to false anyways so that behavior was not being used.

From a infrastructure creation standpoint did not run into any issues. Testing a Viya deployment on the cluster is still in progress.

Reply:
Did you check the [breaking changes](https://github.com/hashicorp/terraform-provider-kubernetes/releases/tag/v2.0.0) section in the release notes for 2.0.0?

Reply:
@thpang I did check for any breaking changes in the 2.0.0, see my previous comment. The `load_config_file` was removed in our code since it was deprecated in this new version.